index.html

```diff
@@ -0,0 +1,339 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
- `required`
- **Well done so far👏👏**
- Please ensure that the code review body appears above 1️⃣ the inline comments to summarize and gives an overall idea about the review ✨, to make it possible, please refer to [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

- I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.
-  I also strongly suggest that in order to sound polite you need to be using words like `please`, `kindly`, `I suggest`, and `I highly recommend ...etc, and many other words you find properly.
- Please explain the **why** you are asking for that change by providing examples and best practices approach ✔️  so the student can learn something 👨‍🏫😄 as well and don't feel the change was unfair 👌.